Makefile

```
@@ -50,6 +50,7 @@ submodules-update:
 	@git submodule update --init --recursive --remote --merge
 
 build:
+	sudo apt -y install unzip
```

(Nitpick) We don't really need `unzip` if we just remove lines 58 and 59:
```
	@mkdir -p ${CHARM_BUILD_DIR}/${CHARM_NAME}
	@unzip ${PROJECTPATH}/${CHARM_NAME}.charm -d 
```
The charms works fine and the tests pass even without this unzipping.

However, if we do indeed keep this, this line should be 
```suggestion
	sudo apt update && sudo apt -y install unzip
```

---

I would like to keep this `unzip` because I try not to change the original code as possible.

BTW `apt update` is already in the action so I don't want to have that here.